### PR TITLE
Add Kumar Karan's string tokenizer program contribution(first)

### PR DESCRIPTION
**Pull request by BharatiyaKaran**

### CODE REVIEW

File: strings/c-or-cpp/strtok.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+* Creation Date : 26-01-2021
+
+* Last Modified : Fri Feb  5 18:41:14 2021
+
+* Created By : Kumar Karan
+
+* Problem Description :
+
+_._._._._._._._._._._._._._._._._._._._._.*/
+
```
Comment (ming-tsai) [inline strings/c-or-cpp/strtok.cpp:16]: Please delete theses line, we're git for manage the history of file

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Please add this file to readme.md index

File: strings/c-or-cpp/strtok.cpp (hunk 1 above)
Comment (BharatiyaKaran) [reply strings/c-or-cpp/strtok.cpp]: Ok thanks Ming, done !

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @BharatiyaKaran,
Could you rename file with complete name, according to contributions guidelines?

**Review by ming-tsai — APPROVED** [review]
Hi @BharatiyaKaran,
Good job, thanks for your contributions, and welcome to our team
> The invitation [link](https://github.com/MakeContributions)

**Comment by BharatiyaKaran** [comment]
Thanks @ming-tsai !

**Pull request merged**